src/olympia/lib/settings_base.py

```diff
@@ -1665,6 +1665,10 @@ def get_redis_settings(uri):
 
     # We don't allow refreshes, instead we simply have a long duration.
     'JWT_ALLOW_REFRESH': False,
+
+    # Prefix for non-apikey jwt tokens. Should be different from 'JWT' which we
+    # already used for api key tokens.
+    'JWT_AUTH_HEADER_PREFIX': 'JWT-i',
```

Comment:
It seems like `Bearer` is the most often used and it is referenced on jwt.io [1]. If we need to detect the type of token I'd imagine we'd put something in the payload rather than changing the type of the authorization (unless we moved away from JWT of course).

[1] https://jwt.io/introduction/#how-do-json-web-tokens-work-